src/pages/BirthDay.jsx

```diff
@@ -1,113 +1,468 @@
+        day: "23일",
+        name: "박유리",
+      },
+      {
+        day: "29일",
+        name: "하유진",
+      },
+      {
+        day: "30일",
+        name: "국다혜",
+      },
+    ],
   },
   {
-    key: 6,
+    id: 6,
     label: "7월",
+    list: [
+      {
+        day: "02일",
+        name: "유지원",
+      },
+      {
+        day: "03일",
+        name: "선승훈",
+      },
+      {
+        day: "06일",
+        name: "윤태호",
+      },
+      {
+        day: "07일",
+        name: "윤동하",
+      },
+      {
+        day: "10일",
+        name: "이재연",
+      },
+      {
+        day: "20일",
+        name: "신중석",
+      },
+      {
+        day: "21일",
+        name: "박한빈",
+      },
+      {
+        day: "28일",
+        name: "윤지원",
+      },
+    ],
   },
   {
-    key: 7,
+    id: 7,
     label: "8월",
+    list: [
+      {
+        day: "03일",
+        name: "윤소영",
+      },
+      {
+        day: "12일",
+        name: "정지민",
+      },
+      {
+        day: "13일",
+        name: "김승찬",
+      },
+      {
+        day: "16일",
+        name: "백승도",
+      },
+      {
+        day: "21일",
+        name: "전대현",
+      },
+      {
+        day: "27일",
+        name: "김민준",
+      },
+      {
+        day: "28일",
+        name: "차미진",
+      },
+    ],
   },
   {
-    key: 8,
+    id: 8,
     label: "9월",
+    list: [
+      {
+        day: "04일",
+        name: "김태균",
+      },
+      {
+        day: "04일",
+        name: "신중범",
+      },
+      {
+        day: "18일",
+        name: "김다은",
+      },
+      {
+        day: "21일",
+        name: "김예빈",
+      },
+      {
+        day: "25일",
+        name: "윤동주",
+      },
+      {
+        day: "28일",
+        name: "마관영",
+      },
+    ],
   },
   {
-    key: 9,
+    id: 9,
     label: "10월",
+    list: [
+      {
+        day: "04일",
+        name: "정구연",
+      },
+      {
+        day: "17일",
+        name: "최대한",
+      },
+      {
+        day: "22일",
+        name: "김옥주",
+      },
+      {
+        day: "26일",
+        name: "황수정",
+      },
+      {
+        day: "31일",
+        name: "김윤주",
+      },
+    ],
   },
   {
-    key: 10,
+    id: 10,
     label: "11월",
+    list: [
+      {
+        day: "07일",
+        name: "정지혁",
+      },
+      {
+        day: "11일",
+        name: "서다현",
+      },
+      {
+        day: "13일",
+        name: "황명표",
+      },
+      {
+        day: "20일",
+        name: "김태경",
+      },
+      {
+        day: "21일",
+        name: "이규영",
+      },
+      {
+        day: "22일",
+        name: "정한슬",
+      },
+    ],
   },
   {
-    key: 11,
+    id: 11,
     label: "12월",
+    list: [
+      {
+        day: "13일",
+        name: "구민경",
+      },
+      {
+        day: "22일",
+        name: "김예슬",
+      },
+    ],
   },
 ];
 
-const ChipItem = styled(Stack)`
-  width: calc(100%/7);
-  height: 25%;
-  margin: 1%;
+const BirthDay = () => {
+  const [showListId, setShowListId] = React.useState(0);
+  const [showList, setShowList] = React.useState(chipData);
```

Comment:
```suggestion
  const [showList, setShowList] = React.useState(chipData[0]);
```
처음에 1월껄로 하면될 것 같은데?

Reply:
```suggestion
  const [birthDayInfo, setBirthDayInfo] = React.useState(chipData[0]);
```
이런식으로 객체를 가지고 있는게 나을 것 같음